doc/btp_gap.txt

```diff
@@ -892,6 +892,189 @@ Commands and responses:
 
                 In case of an error, the error response will be returned.
 
+	Opcode 0x34 - Configure Advertising Set
+
+		Controller Index:	<controller id>
+		Command parameters:	Advertising Set ID (1 octet)
+					SID (1 octet)
+					Own Address Type (1 octet)
+					Advertising Type (1 octet)
+					Primary Advertising PHY (1 octet)
+					Secondary Advertising PHY (1 octet)
+					Secondary Advertising Maximum Skip (1 octet)
+					Coded PHY Options (1 octet)
+					Filter Accept List (1 octet)
+					Directed Advertising (1 octet)
+					Options (2 octets)
```

Comment:
I'd suggest to combine coded phy options, filer accept list options, directed options and options into single 32bit options (or flags) that would allow to tune IUT behavior,  this way we could extend BTP more easily if eg new options (eg CTE) or PHYs are introduced  (as mentioned, BTP doesn't have to map 1-1 to HCI or specific IUT apis)